src/pages/home/report/ReportActionsView.tsx

```diff
@@ -297,53 +307,92 @@ function ReportActionsView({
-        if (!!network.isOffline || isLoadingOlderReportActions || isLoadingInitialReportActions) {
-            return;
-        }
+    const loadOlderChats = useCallback(
+        (force = false) => {
+            // Only fetch more if we are neither already fetching (so that we don't initiate duplicate requests) nor offline.
+            if (
+                !force &&
+                (!!network.isOffline ||
+                    isLoadingOlderReportActions ||
+                    // If there was an error only try again once on initial mount.
+                    (didLoadOlderChats.current && hasLoadingOlderReportActionsError) ||
+                    isLoadingInitialReportActions)
+            ) {
+                return;
+            }
 
-        // Don't load more chats if we're already at the beginning of the chat history
-        if (!oldestReportAction || hasCreatedAction) {
-            return;
-        }
+            // Don't load more chats if we're already at the beginning of the chat history
+            if (!oldestReportAction || hasCreatedAction) {
+                return;
+            }
 
-        if (!isEmptyObject(transactionThreadReport)) {
-            // Get newer actions based on the newest reportAction for the current report
-            const oldestActionCurrentReport = reportActionIDMap.findLast((item) => item.reportID === reportID);
-            Report.getNewerActions(oldestActionCurrentReport?.reportID ?? '0', oldestActionCurrentReport?.reportActionID ?? '0');
+            didLoadOlderChats.current = true;
 
-            // Get newer actions based on the newest reportAction for the transaction thread report
-            const oldestActionTransactionThreadReport = reportActionIDMap.findLast((item) => item.reportID === transactionThreadReport.reportID);
-            Report.getNewerActions(oldestActionTransactionThreadReport?.reportID ?? '0', oldestActionTransactionThreadReport?.reportActionID ?? '0');
-        } else {
-            // Retrieve the next REPORT.ACTIONS.LIMIT sized page of comments
-            Report.getOlderActions(reportID, oldestReportAction.reportActionID);
-        }
-    }, [network.isOffline, isLoadingOlderReportActions, isLoadingInitialReportActions, oldestReportAction, hasCreatedAction, reportID, reportActionIDMap, transactionThreadReport]);
+            if (!isEmptyObject(transactionThreadReport)) {
+                // Get newer actions based on the newest reportAction for the current report
+                const oldestActionCurrentReport = reportActionIDMap.findLast((item) => item.reportID === reportID);
+                Report.getNewerActions(oldestActionCurrentReport?.reportID ?? '0', oldestActionCurrentReport?.reportActionID ?? '0');
 
-    const loadNewerChats = useCallback(() => {
-        if (isLoadingInitialReportActions || isLoadingOlderReportActions || network.isOffline || newestReportAction.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) {
```

Comment:
I noticed this was checking `isLoadingOlderReportActions` which I assume was an error since this is `loadNewerChats`, but maybe someone can confirm that this change is ok.